packages/did-core-test-server/suites/did-spec/defaultSuiteConfig.json

```diff
@@ -1,6 +1,49 @@
 {
   "name": "did-spec",
   "didMethods": {
+    "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar": {
+      "supportedContentTypes": [
+        "application/did+ld+json"
+      ],
+      "dids": ["did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"],
+      "didParameters": {},
+      "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH": {
+        "application/did+ld+json": {
+          "didDocument": {
+            "@context": ["https://www.w3.org/ns/did/v1"],
+            "id": "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH",
+            "verificationMethod": [{
+              "id": "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH",
+              "type": "Ed25519VerificationKey2018",
+              "controller": "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH",
+              "publicKeyBase58": "B12NYF8RrR3h41TDCTJojY59usg3mbtbjnFs7Eud1Y6u"
+            }],
+            "authentication": [
+              "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+            ],
+            "assertionMethod": [
+              "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+            ],
+            "capabilityDelegation": [
+              "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+            ],
+            "capabilityInvocation": [
+              "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+            ],
+            "keyAgreement": [{
+              "id": "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#zBzoR5sqFgi6q3iFia8JPNfENCpi7RNSTKF7XNXX96SBY4",
+              "type": "X25519KeyAgreementKey2019",
+              "controller": "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH",
+              "publicKeyBase58": "JhNWeSVLMYccCk7iopQW4guaSJTojqpMEELgSLhKwRr"
+            }]
+          },
+          "didDocumentMetaData": {
+            "content-type": "application/did+ld+json"
```

Comment:
This is DID Resolution Metadata, not DID Document Metadata.

Reply:
Also, it should be `contentType`, not `content-type`.

Reply:
This fix is being dealt with in PR #37.